src/components/App/App.module.css

```diff
@@ -1,9 +1,13 @@
 .wrap {
     display: block;
     margin: auto;
-    width: 300px;
+    width: 500px;
+    background-color: #ffffff;
+    border-radius: 7px;
+    box-shadow: 0 0 8px rgba(0,0,0,0.3);
 }
 
-.title {
-    font-family: 'Montserrat', sans-serif;
-}
\ No newline at end of file
+.content {
+    text-align: center;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/todo_test/pullRequest?prid=5693537)